### PR TITLE
NavBar, Footer, and Home modification

### PR DESCRIPTION
**Pull request by georgia1515**

### CODE REVIEW

**Review by LeonZhang7 — APPROVED** [review]
Nicely done, approved !

File: ecommerce-website/src/components/Footer.js [hunk 1]
```diff
@@ -3,12 +3,21 @@ import './Footer.css'
...
+          {/*<img
+                alt=""
+                src={require('./../images/heaglw-logo-transparent-v2.png')}
+                width="100"
+                height="100"
+                className="heagle-logo"
+              />{' '}
+          */}
+          </h3> 
+          <p className='heagle-content'>Content about Heagle.</p>
```
Comment (Its-Alann) [inline ecommerce-website/src/components/Footer.js:20]: I added the logos, you should be able to uncomment this

File: ecommerce-website/src/components/Navbar.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
-            <Navbar.Brand href="#">Heagle Logo</Navbar.Brand>
+            <Navbar.Brand href='https://heagle.herokuapp.com/'>
+            {/*<img
+                alt="Logo"
+                src={require('./../images/heaglw-logo-transparent-v2.png')}
+                width="100"
+                height="100"
+                className="heagle-logo"
+            />*/}
+            </Navbar.Brand>
```
Comment (Its-Alann) [inline ecommerce-website/src/components/Navbar.js:20]: Same thing with this, I added the logos

File: ecommerce-website/src/components/products/IndividualProductPage.js [hunk 3]
```diff
@@ -19,7 +19,9 @@ const IndividualProductPage = ({ match }) => {
   useEffect(()=> {
       const getProductFromServer = "https://heagle-backend.herokuapp.com/getProduct/" + pageId.id
       Axios.get(getProductFromServer).then((response) => {
-          setName(response.data[0].name);
+      // Use this const below for when you code in local, comment it out once done
+      //const getProductFromServer = "http://localhost:3001/getProduct/" + pageId.id
```
Comment (Its-Alann) [inline ecommerce-website/src/components/products/IndividualProductPage.js:23]: Please move these two lines above the Axios.get line

**Review by Its-Alann — CHANGES_REQUESTED** [review]
Some files require some changes, logos are on the main branch. The rest looks good thanks.

**Comment by georgia1515** [comment]
The changes have been made @Its-Alann.

**Review by Its-Alann — APPROVED** [review]
Looks good, thanks Georgia!

**Pull request merged**